### DISCUSSION TITLE
What's the new equivalent of editing `node_modules`?

### DISCUSSION THREAD

**Discussion by danieltroger** [yarnpkg/berry#3363] [Q&A] [answered]
To enable transpilation of node_modules with parcel I used to do this `gsed -i 's/!filePath.includes(NODE_MODULES);/true;\ \/\/changemeback/' node_modules/\@parcel/core/lib/summarizeRequest.js` but now with this new fancy zipping thing it has become much more annoying:
1. find the right zip
2. unzip it
3. edit
4. rezip
5. put back
6. pray that yarn doesn't do any signature checks (`yarn run` seems not to do so it's good for at least one time)

I wanted to ask if there's some easy way you have thought of to accomplish this or if I really need to script editing a .zip?

**Comment by 7rulnik** [comment, marked as answer]
https://yarnpkg.com/cli/unplug

Also you can apply patches to packages via https://yarnpkg.com/cli/patch

**Reply by danieltroger** [reply]
Thanks! I'll try unplug for now since it seems to be the easiest solution

**Comment by Embraser01** [comment]
You could also use the patch protocol 
https://yarnpkg.com/features/protocols/#patch

**Reply by danieltroger** [reply]
Damn that patch thing is wild it did exactly what I wanted
